messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/token/store/jpa/JpaTokenStore.java

```diff
@@ -72,29 +70,29 @@ public class JpaTokenStore implements TokenStore {
     private static final String SEGMENT_PARAM = "segment";
 
     private final EntityManagerProvider entityManagerProvider;
-    private final Serializer serializer;
+    private final Converter converter;
     private final TemporalAmount claimTimeout;
     private final String nodeId;
     private final LockModeType loadingLockMode;
 
     /**
      * Instantiate a {@link JpaTokenStore} based on the fields contained in the {@link JpaTokenStoreConfiguration}.
      * <p>
-     * Will assert that the {@link EntityManagerProvider}, {@link Serializer} and {@link JpaTokenStoreConfiguration} are
-     * not {@code null}, otherwise an {@link AxonConfigurationException} will be thrown.
+     * Will assert that the {@link EntityManagerProvider} and {@link JpaTokenStoreConfiguration} are not {@code null},
```

Comment:
We're missing the description of the `Converter` here. Both in the body as the `@param` mention.